### ISSUE TITLE
Missing getters

### ISSUE THREAD

**Issue by fredRos** [edankwan/penis.js#9]
Thanks for creating this awesome package! I wanted to use it right away but quickly ran into problems. Seems like this project is still on the rise so I wanted to add suggestions for improvement:

1. If there is `setHead` how come there is no `getHead`? To many users, this is a very important method. Please consider implementing it, but only after thorough testing.
2. Maybe I was missing the point of the package but why is there no `penis.getLength`? The documentation states that this is about *comparing variables* and how can you do a `penis.js` comparison without the length? Seriously, you should go back and rub your `head`, then upload a new version.

**Comment by edankwan** [comment]
Good idea man! But for the `penis.getLength()`, what should be the return value? Also, do you think `getSize()` would be a better name?

**Comment by pappacena** [comment]
I've heard that `getSize()` doesn't matter.

**Comment by renatoferreirarenatoferreira** [comment]
Yes, it does! It is only told by people who have uglyfied packages.

**Comment by rheber** [comment]
For values which can be compared numerically, `getSize()` could be their absolute difference. Visualise the balls and head at points on a number line.

**Comment by DmitriyPast** [comment]
Not really. the value matters the most. The length is important but it's not the most important.

**Comment by renatoferreirarenatoferreira** [comment]
Guys! This is not aging well...
